### PR TITLE
Correct output_padding for quantized tconv (torch->onnx)

### LINKED ISSUE #104206: [torch->onnx] output_padding is not respected in ConvTransposeNd
### 🐛 Describe the bug

When exporting a quantized ConvTransposeNd to ONNX, the `output_padding` is not respected and instead all-zeros is always used. This is a bug I introduced myself in #102759 (after thinking this parameter only existed on the ONNX side). A fix is incoming.

### Versions

Not platform specific.

### PR DESCRIPTION
**Pull request by gustavla**
- In #102759, the support for `quantized::conv_transposeNd` was introduced. This incorrectly set `output_padding` to all zeros. Turns out, you can specify output_padding in PyTorch, but this parameter was not being unpacked correctly and thus did not show up in the python torch->onnx code.
- This adds unpacking of output_padding in `unpack_quantized_weights.cpp` when needed. It also adds this as a parameter in the python functions and uses that (and removes the all-zero defaults)
- Another issue with #102759 is that it only added these new ops to opset10 without adding the ability to specify axis in opset13. This PR also fixes this.

Fixes #104206

### CODE REVIEW

File: torch/onnx/symbolic_opset10.py [hunk 1]
```diff
@@ -958,6 +958,7 @@ def quantized_conv_transpose1d(
     bias,
     stride,
     padding,
+    output_padding,
```
Comment (gustavla) [inline torch/onnx/symbolic_opset10.py:961]: Note sure how to address this linter error. These are not user-facing functions and not feeding in `output_padding` was a bug to begin with.

File: torch/onnx/symbolic_opset13.py [hunk 2]
```diff
@@ -888,9 +888,9 @@ def quantized_linear(
     return symbolic_helper.quantize_helper(g, output, op_scale, op_zero_point)
 
 
-@_onnx_symbolic("quantized::conv2d")
+@_onnx_symbolic("quantized::conv1d_relu")
 @_beartype.beartype
-def quantized_conv2d(
+def quantized_conv1d_relu(
```
Comment (gustavla) [inline torch/onnx/symbolic_opset13.py:893]: This seemingly odd 1d->2d diff is just from copying the order used in `symbolic_opset10.py`. This `quantized_conv2d` is unchanged and shows up below.

File: torch/onnx/symbolic_opset10.py (hunk 1 above)
Comment (BowenBao) [reply torch/onnx/symbolic_opset10.py:961]: I was confused for a moment how this syntactically worked, as I was assuming the C++ `quantized::conv_transpose1d` signature did not change. But later realized the assumption is not true.

File: torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp [hunk 3]
```diff
@@ -571,10 +596,14 @@ void unpackQuantizedWeightsHelper(
 
     // add conv arguments: stride, padding, dilation, groups
```
Comment (BowenBao) [inline torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp:597]: nit: sync comment

**Review by BowenBao — APPROVED** [review]
LGTM, thanks for fix!

**Comment by BowenBao** [comment]
Please revert accidental third_party updates and we are good to go!

**Comment by gustavla** [comment]
@bowenbao My bad! The accidental third_party updates are gone now. Thanks!

**Pull request closed without merging**